src/test/java/com/telefonica/iot/cygnus/sinks/OrionHDFSSinkTest.java

```diff
@@ -23,7 +23,7 @@
 import static org.junit.Assert.*; // this is required by "fail" like assertions
```

Comment:
The same answer than "OrionCKANSink", this string must be configured by the users in a Cygnus parameter and it would be very disturbing to change it now. Let's live with it!